### PR TITLE
[wmco] Refactor Windows service start/creation

### PR DESCRIPTION
**Pull request by saifshaikh48**
This change was made to reduce code duplication in the creation and start of Windows services, as well as when checking if a particular file exists on the VM. This makes it easier when adding new services into Windows VM configuration.

### CODE REVIEW

File: pkg/controller/windowsmachine/windows/windows.go [hunk 1]
```diff
@@ -146,6 +148,15 @@ func (vm *windows) CopyFile(filePath, remoteDir string) error {
 	return nil
 }
 
+func (vm *windows) FileExists(path string) (bool, error) {
+	out, err := vm.Run("Test-Path "+path, true)
+	if err != nil {
+		return false, errors.Wrapf(err, "error checking if file %s exists on Windows VM %s. output: %s",
+			path, vm.ID(), out)
```
Comment (aravindhp) [inline pkg/controller/windowsmachine/windows/windows.go:155]: No need to return `out` in the error. I will be logging that error in #147 and that should suffice.

**Review by aravindhp — CHANGES_REQUESTED** [review]

**Comment by LorbusChris** [comment]
/approve
I suggest we merge this asap so I can easily base further windows_exporter work on top.

**Review by aravindhp — APPROVED** [review]
/approve

Thanks for working on this, @saifshaikh48

**Pull request merged**